src/localplayer.cpp

```diff
@@ -1147,9 +1147,9 @@ void LocalPlayer::handleAutojump(f32 dtime, Environment *env,
 
 	// see if we can get a little bit farther horizontally if we had
 	// jumped
-	v3f run_delta = position_before_move - m_position;
+	v3f run_delta = initial_position - m_position;
 	run_delta.Y = 0.0f;
-	v3f jump_delta = position_before_move - jump_pos;
+	v3f jump_delta = initial_position - jump_pos;
```

Comment:
Might as well swap the terms too, i'm +1 if you do.

Reply:
For a 1D example:
Moving from 2 to 5 is a movement of +3, which is '5 - 2', 'final - initial'.
These deltas are wrong (inverted), they only work because getting the length of a vector is unaffected by the sign.

Reply:
Whether it's correct or wrong depends on the viewpoint. Deltas can be measured in all directions.

Reply:
Somewhat true, but a delta is usually measured going forwards in time from initial to final position.